File: PATTERNS/Pattern38/Pattern38.py
def print_pattern_38(n):
    n *= 2
    for i in range(0, n):
        output = ""
        for j in range(0, n):
            if j == 0 or j == i or j == n - i - 1 or j == n - 1:
                output += "*"
            else:
                output += " "
        print(output)

n = int(input("enter a number : "))
print_pattern_38(n)

File: PATTERNS/Pattern38/Pattern38.go
package main
import "fmt"

func printPattern(n int) {
	// Multiply n by 2 to determine the size of the pattern
	n = n * 2
	for row := 1; row <= n; row++ {
		for col := 1; col <= n; col++ {
			// Check conditions to determine if a "*" or " " should be printed
			if row == col || col+row == n+1 || col == 1 || col == n {
				fmt.Print("*")
			} else {
				fmt.Print(" ")
			}
		}
		fmt.Println() // Move to the next line after printing each row
	}
}

func main() {
	var num int
	fmt.Print("Enter the number of times you want to execute: ")
	fmt.Scanln(&num)
	printPattern(num)
}
